runtime/shibuya/src/lib.rs

```diff
@@ -1333,12 +1333,28 @@ impl OnRuntimeUpgrade for DynamicEvmBaseFeeMigration {
     }
 }
 
+/// Simple `OnRuntimeUpgrade` logic to remove all corrupted mappings
+/// after the fixing the typo in mapping names in pallet.
+pub struct ClearCorruptedUnifiedMappings;
+impl OnRuntimeUpgrade for ClearCorruptedUnifiedMappings {
+    fn on_runtime_upgrade() -> Weight {
+        let maybe_limit = pallet_unified_accounts::EvmToNative::<Runtime>::iter().count();
+        let total_w = maybe_limit as u64 * 2;
+        // remove all items
+        let _ = pallet_unified_accounts::EvmToNative::<Runtime>::clear(maybe_limit as u32, None);
+        let _ = pallet_unified_accounts::NativeToEvm::<Runtime>::clear(maybe_limit as u32, None);
+
+        <Runtime as frame_system::Config>::DbWeight::get().writes(total_w)
```

Comment:
`iter().count()` will read everything, so you also need to account for that in the weight.

`clear` will also read keys, AFAIK, adding up to the weight.
So `read_writes(total, total)`